op-node/rollup/superchain.go

```diff
@@ -108,7 +108,7 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 		RegolithTime:           &regolithTime,
 		CanyonTime:             superChain.Config.CanyonTime,
 		DeltaTime:              deltaTime,
-		EcotoneTime:            superChain.Config.EcotoneTime,
+		EcotoneTime:            nil, // Ecotone upgrade was vetoed
```

Comment:
The veto would _only_ apply to OP Mainnet (id 10), not other mainnet chains.

Reply:
Of course. Fixed it [3c4a3ee](https://github.com/ethereum-optimism/optimism/pull/9778/commits/3c4a3ee5a25ef7f9145ff5919987212359b00717)